src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService userService;
+
+    public AuthenticationService() {
+        userService = new UserService();
+    }
+
     public boolean login(String email, String password) {
-        return false;
+        return Arrays.stream(UserService.getUsers())
+                .anyMatch(user -> user.getEmail().equals(email)
+                        && user.getPassword().equals(password));
```

Comment:
you can reuse existing logic
public boolean login(String email, String password) {
    User user = UserService.findByEmail(email);
    return user != null && user.getPassword().equals(password);
}